integration-tests/micrometer-prometheus/pom.xml

```diff
@@ -42,6 +42,12 @@
             <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
         </dependency>
 
+        <!-- Verify that the client works with OpenTracing -->
```

Comment:
Not sure I agree with adding OpenTracing to the Micrometer Prometheus integration test.

Short/Medium term this will likely require testing with OpenTelemetry when we've integrated the two.

Reply:
Well, I don't want to add 5 extra minutes to CI just to make sure that a client request doesn't blow up in native mode.

I can certainly do it, but if we go down that road with every such case, CI times will increase fast

Reply:
I think I would lean towards not testing this, even if not ideal.

Reply:
OK, removed